components/authz-service/testhelpers/testhelpers.go

```diff
@@ -157,6 +158,44 @@ func (tf *TestFramework) Shutdown(t *testing.T, ctx context.Context) {
 	// this TestFramework's authz instance could write some bad state while the next test is running.
 }
 
+// SetupProjectsAndRulesWithDB is a simplified test framework
+// useful for integration tests with just the DB.
+func SetupProjectsAndRulesWithDB(t *testing.T) (
```

Comment:
Reinstated this simpler helper that allows focussing just on the DB for these tests.